neuro_extras/cli.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+import click
+
+from .version import __version__
+
+
+class ClickLogHandler(logging.Handler):
+    def emit(self, record: logging.LogRecord) -> None:
+        try:
+            msg = self.format(record)
+            click.echo(msg)
+        except Exception:
+            self.handleError(record)
+
+
+@click.group()
+@click.version_option(
+    version=__version__, message="neuro-extras package version: %(version)s"
+)
+def main() -> None:
```

Comment:
Maybe let's move it into main.py for the sake of simplicity? Or we need to keep the main group in a separate file? (IDK whether we could/are planning to have several root CLI groups)

Reply:
We need to keep it in a separate file to avoid circular dependencies. Maybe there's a better way to handle this, but I couldn't find an alternative.

Reply:
Ok, got you! tbh, I didn't found the circle in imports and would be grateful if you could point it out as for a blind guy 😄 
In any case, all is LGTM, so let's proceed

Reply:
Sure, I can demostrate:
* `__init__.py`: `from .main import main`
* `main.py`: `from .config import config_save_docker_json`
* `config.py`: `from .main import main`

Basically, in this case `main.py` has to import other modules in order to get commands for `click`, whereas other modules import `main` in order to get `@main` annotation.

Reply:
Oh, now I see it!
Thanks for explanation!